libs/@local/hash-isomorphic-utils/src/entity-store-plugin.ts

```diff
@@ -375,10 +372,34 @@ const entityStoreReducer = (
           draftState.trackedActions.push({ action, id: uuid() });
         }
 
+        const now = new Date().toISOString();
+
         draftState.store.draft[action.payload.draftId] = {
           metadata: {
+            archived: false,
             recordId: {
               entityId: action.payload.entityId,
+              editionId: now,
```

Comment:
Just for my understanding, is the `editionId` always a timestamp? Or is it just a timestamp in the context of the entity store, and a UUID otherwise?

Reply:
In subgraphs it is always a timestamp, corresponding to the start of the decision time range the edition applies over